frontend/src/component/changeRequest/ChangeRequest/ChangeRequest.tsx

```diff
@@ -29,11 +29,19 @@ export const ChangeRequest: VFC<IChangeRequestProps> = ({
                 }
             />
 
-            {changeRequest.segments?.map(segment => (
+            {changeRequest.segments?.map(segmentChange => (
                 <SegmentChange
-                    key={segment.payload.id}
-                    segmentChange={segment}
+                    key={segmentChange.payload.id}
+                    segmentChange={segmentChange}
                     onNavigate={onNavigate}
+                    actions={
+                        <ChangeActions
+                            changeRequest={changeRequest}
+                            feature={segmentChange.name}
```

Comment:
Hmm. segmentChange.name is not a feature name, but a current segment name of the segment you're changing.

Reply:
Yes, currently ChangeActions has actions for delete and edit. Edit needs a name of something. Currently providing segment name, but when we implement edit of segment change, then we need to refactor this a little.